src/test/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSinkUnitTest.java

```diff
@@ -18,8 +19,8 @@
 
 
 public class ReadsSparkSinkUnitTest extends BaseTest {
-    @DataProvider(name = "loadReads")
-    public Object[][] loadReads() {
+    @DataProvider(name = "loadReadsBAM")
+    public Object[][] loadReadsBAM() {
         String dir = "src/test/resources/org/broadinstitute/hellbender/tools/BQSR/";
```

Comment:
Please switch this to use the source directory pattern found in other test files (publicTestDir). Also, use this path
src/test/resources/org/broadinstitute/hellbender/engine/dataflow/ReadsSource/HiSeq.1mb.1RG.2k_lines.bam